swan-lake/resources/featured-scenarios/create-your-first-connector-with-ballerina.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: ballerina-create-your-first-connector-left-nav-pages-swanlake
+title: Create your first connector with Ballerina
+description: step-by-step guide to create Ballerina connectors using OpenAPI specifications.
+keywords: ballerina, connector, library, REST, API, openAPI, Ballerina Central
+permalink: /learn/create-your-first-connector-with-ballerina/
+active: create-your-first-connector-with-ballerina
+intro: This guide walks you through creating your first Ballerina connector using an OpenAPI specification.
+---
+
+## Introduction
+
+Ballerina is a programming language designed for smooth integration, offering a large library of pre-built connectors. These connectors are special packages that make it easy to communicate with external services, usually through REST APIs. By using connectors, developers can quickly integrate third-party services into their Ballerina applications without having to worry about the technical details of API interactions.
+
+Along with its powerful library ecosystem, Ballerina also allows developers to easily create, share, and manage their own custom connectors. These connectors can be published on Ballerina Central, making them available for the entire community to use and reuse in their projects.
+
+In this guide, we'll walk you through how to generate your first Ballerina connector using an OpenAPI specification. This is one of the fastest and easiest ways to build connectors, helping you quickly integrate external services into your Ballerina projects.
+
+## Set up the prerequisites
+
+Before we begin, make sure you have:
+
+1. A basic understanding of Ballerina SwanLake and the [latest version installed](https://ballerina.io/downloads/).
+2. An OpenAPI specification of the API for which you’re building the connector, along with API credentials (e.g., Twitter Developer account).
+3. A GitHub account, and Git installed locally.
+4. Visual Studio Code with the [Ballerina extension](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina).
+
+## Step 1: Set the project structure
+
+1. Create a new GitHub repository with an appropriate name. For Ballerina official connectors, the repository name will follow the pattern: `module-ballerinax-<connector-name>` (e.g., `module-ballerinax-twitter`). But for custom connectors, you can choose a name that suits your connector.
+
+2. Clone your newly created empty repository to your local machine:
+   ```bash
```

Comment:
```suggestion
   ```
```
Remove bash to avoid incorrect syntax highlighting.

Reply:
Addressed in https://github.com/ballerina-platform/ballerina-dev-website/pull/9434/commits/7f3e76648db2831c1d956d733e403ad1999cc40d